.github/workflows/test-content-schemas-1.yml

```diff
@@ -0,0 +1,113 @@
+name: Test Content Schemas (1)
+
+on:
+  workflow_dispatch: {}
+  push:
+    branches:
+      - main
+    paths:
+      - "content_schemas/**"
+      - "!content_schemas/README.md"
+  pull_request:
+    paths:
+      - "content_schemas/**"
+      - "!content_schemas/README.md"
+
+jobs:
+  test-collections-publisher:
+    name: Test Collections Publisher
+    uses: alphagov/collections-publisher/.github/workflows/rspec.yml@main
+    with:
```

Comment:
optional: worth using a YAML anchor to factor out the repeated block?

Reply:
Good shout! Unfortunately, at the moment, [workflows don't support YAML anchors](https://github.com/actions/runner/issues/1182) 😞

Reply:
Ohh sorry hadn't come across that. Thanks!